atlas-starter.py

```diff
@@ -0,0 +1,91 @@
+import pymongo
+
+# Replace the placeholder data with your Atlas connection string. Be sure it includes
+# a valid username and password! Note that in a production environment,
+# you should not store your password in plain-text here.
+
+client = pymongo.MongoClient(
+      "mongodb+srv://<username>:<password>@<cluster-name>/test?retryWrites=true&w=majority")
+
+# use a database named "my_database"
+
+db = client.my_database
+
+# use a collection named "recipes"
+
+my_collection = db["recipes"]
+
+recipe_documents = [{ "name": "elotes", "ingredients": ["corn", "mayonnaise", "cotija cheese", "sour cream", "lime"], "prep_time": 35 },
+                    { "name": "loco moco", "ingredients": ["ground beef", "butter", "onion", "egg", "bread bun", "mushrooms"], "prep_time": 54 },
+                    { "name": "patatas bravas", "ingredients": ["potato", "tomato", "olive oil", "onion", "garlic", "paprika"], "prep_time": 80 },
+                    { "name": "fried rice", "ingredients": ["rice", "soy sauce", "egg", "onion", "pea", "carrot", "sesame oil"], "prep_time": 40 }]
+
+# drop the collection in case it already exists
+
+my_collection.drop()  
+
+# INSERT DOCUMENTS
+#
+# You can insert individual documents using collection.insert_one().
+# In this example, we're going to create four documents and then 
+# insert them all with insert_many().
+
+my_collection.insert_many(recipe_documents)
```

Comment:
Our other examples extract the output of insert_many and print a message so the user knows that something was done. That might be a good idea here too! It also might be helpful to handle exceptions/errors gracefully here in case users have something wonky with their permissions in their Atlas cluster.

Example:
https://github.com/mongodb-university/atlas_starter_java/blob/master/src/main/java/mongodb/Main.java#L95

Reply:
Good point. Updated the code to return the number of docs inserted, and added error logic if the user doesn't have write permissions.